### PR TITLE
[sca, fpga] Increase start trigger delay for AES and KMAC peripherals

### PR DESCRIPTION
**Pull request by vogelpi**
During FPGA measurements we have been observing stability issues with the power rails for fixed vs. random TVLA. In some cases, loading the fixed plaintext/key discharges the power rails slightly more or less than loading random plaintexts/keys, which then leads to a vertical offset in the traces and thus "fake" leakage unrelated to the actual processing. Increasing the start trigger delay allows the power rails to recharge before the actual processing phase to be analyzed starts.

### CODE REVIEW

**Review by msfschaffner — APPROVED** [review]

File: sw/device/sca/kmac_serial.c [hunk 1]
```diff
@@ -50,11 +50,13 @@ enum {
...
-   * clock cycles and the scope captures 200 clock cycles at kClockFreqCpuHz
-   * (2000 samples).
+   * and disturb the capture. Currently, we use a start trigger delay of 320
+   * clock cycles and the scope captures 160 clock cycles at kClockFreqCpuHz
+   * (3200 samples). On the scope side, an offset of 395 clock cycles (7900
+   * samples) is used to ignore 1) the start trigger delay for the PROCESS
+   * command, and 2) the absorb phase for the fixed prefix.
    */
-  kIbexSha3SleepCycles = 800,
+  kIbexSha3SleepCycles = 1180,
```
Comment (abdullahvarici) [inline sw/device/sca/kmac_serial.c:59]: We are increasing `kIbexAesSleepCycles` by 280 but `kIbexSha3SleepCycles` by 380 although the change in delay is equal to 280 cycles. What is the reason for that difference?

**Review by abdullahvarici — COMMENTED** [review]
Thanks Pirmin. I only got a question.

File: sw/device/sca/kmac_serial.c (hunk 1 above)
Comment (vogelpi) [reply sw/device/sca/kmac_serial.c:59]: Well spotted @abdullahvarici !

For SHA3/KMAC, the delay is actually observed twice as there are two commands that Ibex sends to the KMAC peripheral. First a `START` to initialize the Keccak core and start loading data from the message FIFO into the Keccak state. Second, a `PROCESS` command to add the padding and kick off the actual Keccak processing. On FPGA, Ibex can send both commands back2back but the command delay is applied to both. So the increase in total delay is actually 2x 280 cycles. 

The total number of sleep cycles for KMAC I estimated as
- 2x the delay = 640 cycles
- Loading key, padding + SHA3 processing + loading the message afterwards= 160 cycles
- 380 spare cycles

Comment (abdullahvarici) [reply sw/device/sca/kmac_serial.c:59]: Thanks @vogelpi for the explanation!

**Pull request merged**